Dfe.Data.SearchPrototype/SearchForEstablishments/SearchContext.cs

```diff
@@ -16,6 +16,8 @@ public sealed class SearchContext
         public string TargetCollection { get; }
 
         /// <summary>
+        /// The following arguments are passed via the constructor and are not changeable
+        /// once an instance is created, this ensures we preserve immutability.
```

Comment:
Instead, can the summary be "Encapsulates the data necessary to perform the search"

Reply:
I think it's useful to state this is immutable IMO, but happy to debate ;)